pkgs/utils/src/extend.zig

```diff
@@ -0,0 +1,18 @@
+const std = @import("std");
+const castFactory = @import("./cast.zig");
+pub usingnamespace castFactory;
+
+pub fn Extend(To: type, from: anytype, with: anytype) To {
```

Comment:
I don't see anything obviously wrong, what is the error that is being produced?

Reply:
will create a separate extend PR and add a testcase which I would love to have to see/debug to make it work